distribution/lib/Standard/Database/0.0.0-dev/src/Dialect.enso

```diff
@@ -293,3 +293,12 @@ default_fetch_primary_key connection table_name =
         selected = keys_table.select_columns ["COLUMN_NAME", "KEY_SEQ"] case_sensitivity=Case_Sensitivity.Insensitive reorder=True
         key_column_names = selected.sort 1 . at 0 . to_vector
         if key_column_names.is_empty then Nothing else key_column_names
+
+## PRIVATE
+type Temp_Table_Style
+      ## PRIVATE
+         The temporary table is created using a create table statement.
+      Temporary_Table
+      ## PRIVATE
+         The temporary table is created using a # table name.
+      Hash_Prefix
```

Comment:
Do we need an `Unsupported` here?

Reply:
I think we add it if we ever find a DB that doesn't support temp tables. Otherwise it is just YAGNI.